en/docs/security-announcements/security-advisories/2025/WSO2-2025-3961.md

```
@@ -0,0 +1,83 @@
+| WSO2 Identity Server                |     5.11.0      |       399       |
+| WSO2 Identity Server                |      6.0.0      |       235       |
+| WSO2 Identity Server                |      6.1.0      |       230       |
+| WSO2 Identity Server                |      7.0.0      |       101       |
+| WSO2 Identity Server as Key Manager |     5.10.0      |       353       |
+| WSO2 Open Banking IAM               |      2.0.0      |       402       |
+| WSO2 Traffic Manager                |      4.5.0      |        2        |
+| WSO2 Universal Gateway              |      4.5.0      |        2        |
+
+
+### CREDITS
+WSO2 thanks, **[Danh Nguyen Hoang]()** for responsibly reporting the identified issue and working with us as we addressed it.
```

markdownlint-cli2 (0.18.1)

No empty links

(MD042, no-empty-links)